### PR TITLE
App workflow for Explorer DEV

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2322

### CODE REVIEW

File: .github/workflows/explorer-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Enable Explorer maintenance on DEV
```
Comment (AndreiNavoichyk) [inline .github/workflows/explorer-dev.yaml:1]: Engineer should be able to select in the Dropdown:
1. App
2. Env

File: .github/workflows/explorer-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      build_container: 'node:14-buster-slim'
+      deploy_container: 'ubuntu:20.04'
+      install_packages_command: 'echo "Installing is not required"'
+      build_command: 'printf "Cere team\n team@cere.network\n" | ./install.sh'
+      path_to_static_files_to_upload: './dist'
+    secrets:
+      NETWORK_AWS_ACCESS_KEY_ID: ${{ secrets.DEV_NETWORK_AWS_ACCESS_KEY_ID }}
+      NETWORK_AWS_SECRET_ACCESS_KEY: ${{ secrets.DEV_NETWORK_AWS_SECRET_ACCESS_KEY }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      S3_BUCKET_NAME: ${{ secrets.EXPLORER_S3_BUCKET_NAME_DEV }}
```
Comment (AndreiNavoichyk) [inline .github/workflows/explorer-dev.yaml:19]: Could we use `DEV` always at the beginning or at the end?

**Review by AndreiNavoichyk — COMMENTED** [review]
@DmitriyMolch 
1. Please, add readme
2. Rename repo to `maintenance-page`

File: .github/workflows/explorer-dev.yaml (hunk 1 above)
Comment (DmitriyMolch) [reply .github/workflows/explorer-dev.yaml:1]: @AndreiNavoichyk, I suggest to run a minimal working version to see how it works and then see how we can optimize it.

File: .github/workflows/explorer-dev.yaml (hunk 2 above)
Comment (DmitriyMolch) [reply .github/workflows/explorer-dev.yaml:19]: At the end seems  consistent with the previous naming convention.

**Comment by DmitriyMolch** [comment]
> @DmitriyMolch
> 
> 1. Please, add readme
> 2. Rename repo to `maintenance-page`

1. We already have readme.
2. Renamed.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**